core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -1229,6 +1224,19 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
     }
   }
 
+  private void unparseDiffFunction(SqlWriter writer, SqlCall call, int leftPrec, int rightPrec,
+      String functionName) {
+    final SqlWriter.Frame date_diff = writer.startFunCall(functionName);
```

Comment:
you can change this frame name from 'date_diff' to diffFunctionFrame or anything common as timestamp_diff and date_diff are calling this method

Reply:
Done.